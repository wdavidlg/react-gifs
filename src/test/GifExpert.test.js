import React from 'react';
import {shallow} from 'enzyme'
import GifExpert from '../GifExpert'

describe('pruebas con <GifExpert />', () => {
    
    test('debe mostrar el componente correctamente', () => {
        const wrapper = shallow(<GifExpert/>)
        expect(wrapper).toMatchSnapshot();
    })

})
