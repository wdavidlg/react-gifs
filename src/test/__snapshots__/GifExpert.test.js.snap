// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas con <GifExpert /> debe mostrar el componente correctamente 1`] = `
<Fragment>
  <h2>
    GifExpert
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <div
    className="categories"
  />
  <div />
</Fragment>
`;
